RPI4: 1.7.9 fails to compile

https://pastebin.com/HEFMfj27
I'm using retropie with raspbian lite on my pi4 to compile. 1.7.8-1.7.8.4 have compiled fine.
``` CC gfx/common/sdl2_common.c
gfx/common/sdl2_common.c: In function ‘sdl2_set_handles’:
gfx/common/sdl2_common.c:70:55: error: ‘union <anonymous>’ has no member named ‘wl’
          video_driver_display_set((uintptr_t)info.info.wl.display);
                                                       ^
gfx/common/sdl2_common.c:71:54: error: ‘union <anonymous>’ has no member named ‘wl’
          video_driver_window_set((uintptr_t)info.info.wl.surface);
                                                      ^
make: *** [Makefile:213: obj-unix/release/gfx/common/sdl2_common.o] Error 1```

--- a/gfx/common/sdl2_common.c
+++ b/gfx/common/sdl2_common.c
@@ -65,7 +65,8 @@ void sdl2_set_handles(void *data, enum rarch_display_type display_type)
 #endif
          break;
       case RARCH_DISPLAY_WAYLAND:
-#ifdef HAVE_WAYLAND
+/* SDL_VIDEO_DRIVER_WAYLAND is defined by SDL2 */
+#if defined(HAVE_WAYLAND) && defined(SDL_VIDEO_DRIVER_WAYLAND)
          video_driver_display_type_set(RARCH_DISPLAY_WAYLAND);
          video_driver_display_set((uintptr_t)info.info.wl.display);
          video_driver_window_set((uintptr_t)info.info.wl.surface);

